src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -113,7 +113,7 @@ public CoreCaseDataService(
         this.jobSchedulerService = jobSchedulerService;
         this.ccdCreateCaseService = ccdCreateCaseService;
         this.caseDetailsConverter = caseDetailsConverter;
-        this.intentionToProceedDeadlineCalculator = intentionToProceedDeadlineCalculator;
+        this.intentionToProceedDeadlineCalculator = stayClaimCalculator;
```

Comment:
intentionToProceedDeadlineCalculator does this need to be StateTransitionCalculator or stayClaimCalculator?